hq_superset/models.py

```diff
@@ -41,6 +51,10 @@ def update_dataset(self):
             raise TableMissing(f'{self.data_source_id} table not found.')
         table = sqla_table.get_sqla_table_object()
 
+        if table.schema and table.schema in SKIP_DATASET_CHANGE_FOR_SCHEMAS:
```

Comment:
Will this then only ignore future dataset changes, or will it also ignore ones that are currently queued?

Reply:
Had an offline sync with Manish.

Reply:
Offlined that the change will apply to any domains added to config and celery restarted.